controllers/release_controller.go

```diff
@@ -113,11 +113,22 @@ func (r *ReleaseReconciler) getReleaseLink(ctx context.Context, release *v1alpha
 }
 
 // getReleaseStrategy loads and returns the ReleaseStrategy referenced in the given ReleaseLink.
-func (r *ReleaseReconciler) getReleaseStrategy(ctx context.Context, releaseLink *v1alpha1.ReleaseLink) (*v1alpha1.ReleaseStrategy, error) {
+func (r *ReleaseReconciler) getReleaseStrategy(ctx context.Context, userReleaseLink *v1alpha1.ReleaseLink) (*v1alpha1.ReleaseStrategy, error) {
```

Comment:
Do we want a `getTargetReleaseLink` function that we can leverage both here and in my work? So we don't duplicate

Reply:
Wouldn't be confusing to allow to pass an org ReleaseLink? It would fail to get the strategy. I'm starting to think if Scott was right about having two different CRDs for this...

Reply:
I'll add a guard for empty/unset strategy.

Reply:
Agreed to the confusion part, but nothing stops us from passing an org `ReleaseLink` anyways. A function is self contained so I don't think we should name the parameters based on what we expect will be passed in (and expectations/requirements are changing fast and often...). The function takes a `ReleaseLink` so I think it should be named just that